File: programmers/Level01/Programmers_12933.java
package programmers.Level01;

import java.util.Arrays;

public class Programmers_12933 {
    public long solution(long n) {
        long answer = 0;

        String[] s = Long.toString(n).split("");
        Arrays.sort(s);
        
        for(int i = s.length-1 ; i >= 0; i--){
            answer = answer * 10 + Integer.parseInt(s[i]);
        }

        return answer;
    }
}

File: programmers/Swift/Level01/Programmers_12933.swift
class Programmers_12933 {
    func solution(_ n:Int64) -> Int64 {
        let array = String(n).map{String($0)}.sorted{$0>$1}
        var answer = String()
        array.map{
            answer += $0
        }
        return Int64(answer)!
    }
}
